### PR TITLE
Main

### PR DESCRIPTION
**Pull request by TymianekPL**
Instead of "Ping" send embed with bot uptime, API latency and bot latency

### CODE REVIEW

**Comment by TymianekPL** [comment]
It's good ngl

File: src/handlers/commands/ping.js [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+          .setDescription("Replies with Pong 🏓"),
 
-    async execute(interaction) {
-        await interaction.reply({ content: 'Pong 🏓', ephemeral: true});
-    },
-};
\ No newline at end of file
+     async execute(interaction) {
+          let uptime = Math.floor(Client.uptime / 1000 / 60 / 60);
+          let sufix = "h";
```
Comment (lostdesign) [inline src/handlers/commands/ping.js:12]: `const` if you are not reassigning

File: src/handlers/commands/ping.js [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+          if (uptime === 0) {
+               uptime = Math.floor(Client.uptime / 1000 / 60);
+               sufix = "min";
+          }
+          if (uptime === 0) {
```
Comment (lostdesign) [inline src/handlers/commands/ping.js:17]: Why do you do the same check again?

File: src/handlers/commands/ping.js [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
+               .setTitle("Ping command")
+               .setTimestamp()
+               .addField("API latency", `${Math.round(Client.ws.ping)}ms`)
+               .addField(
+                    "Bot latency",
+                    `${Math.round(Date.now() - interaction.createdTimestamp)}ms`
+               )
+               .addField("Uptime", `${uptime}${sufix}`)
+               .setFooter("/ping")
+               .setColor("RANDOM");
```
Comment (lostdesign) [inline src/handlers/commands/ping.js:31]: can this be any hex? If so we should use linked's color

**Review by lostdesign — CHANGES_REQUESTED** [review]
Thanks for your PR, overall nice - would it be possible to write functions that handle the suffix automatically? There is already good functions out there or even libraries.

File: src/handlers/commands/ping.js (hunk 1 above)
Comment (TymianekPL) [reply src/handlers/commands/ping.js:12]: I am reassigning sufix

File: index.js [hunk 4]
```diff
@@ -47,4 +47,7 @@ for (const file of eventFiles) {
     }
 }
 
-client.login(token);
\ No newline at end of file
+client.login(token);
+
+// To make ping work
+module.exports.Client = Client;
```
Comment (lostdesign) [inline index.js:53]: instead of exporting the client, the client should be passed into the function

**Review by lostdesign — CHANGES_REQUESTED** [review]

**Comment by lostdesign** [comment]
@TymianekPL thanks for the inspo - quickly added the features myself. Please refer to the CONTRIBUTING.md before opening a PR.

**Pull request closed without merging**